### DISCUSSION TITLE
Parsing a noun chunk

### DISCUSSION THREAD

**Discussion by chrishmorris** [explosion/spaCy#10266] [Help: Coding & Implementations] [answered]
Spacy is awesome, thank you all for your contributions to it.

I am using it to analyse biomedical text. This often contains large noun chunks, e.g. *human malignant breast cancer cell lines*, which Spacy parses as: `[human [malignant [breast [cancer [cell lines]]`. The correct parse is actually `[[[human [malignant [breast cancer]]] cell] lines]`, and the difference matters for my application.

I can generate a large training set automatically. Can I retrain the parser? Or is this behaviour hard coded and not trainable?

**Comment by polm** [comment, marked as answer]
It's possible to train the parser, the same as any other component. We don't have guides for it specifically because the parser model is pretty genre neutral, so most users don't need it, though your case is an exception to that.

Before training your own parser model, I would take a look at existing pretrained models for your field, like [sciSpacy](https://allenai.github.io/scispacy/).
